### PR TITLE
New format in the conditions section of the simulation config file fo…

### PR DESCRIPTION
**Pull request by WeinaJi**
…r synapse variables

### CODE REVIEW

File: source/sonata_simulation.rst [hunk 1]
```diff
@@ -135,17 +135,32 @@ Parameters defining global experimental conditions.
...
    extracellular_calcium           float      Optional    Extracellular calcium concentration. When this parameter is provided, apply it to the synapse uHill parameter to scale the U parameter of synapses (py-neurodamus only feature). If not specified, U is set directly as read from edges file.
    minis_single_vesicle            boolean    Optional    For synapse models with multiple vesicles, limit spontaneous release to single vesicle when true. Default is false which allows for any number of ready vesicles to be potentially released.
    randomize_gaba_rise_time        boolean    Optional    When true, enable legacy behavior to randomize the GABA_A rise time in the helper functions. Default is false which will use a prescribed value for GABA_A rise time.
+   mechanisms                                 Optional    Properties to assign values to variables in synapse MOD files.
+                                                          The format is a dictionary with keys being the SUFFIX of a MOD file (unique name of mechanism) and values being dictionaries of variables' names in the MOD file and their values.
```
Comment (WeinaJi) [inline source/sonata_simulation.rst:143]: In response to a review comment received on GitLab, `SUFFIX` name is not always the same as mechanism/MOD name, but is unique name for the mod. So I add some words here to clarify `SUFFIX`.

Comment (mgeplf) [reply source/sonata_simulation.rst]: I added a link to the nmodl2 doc about suffix, hope that's ok.

**Review by mgeplf — DISMISSED** [review]

File: source/sonata_simulation.rst [hunk 2]
```diff
@@ -135,17 +135,32 @@ Parameters defining global experimental conditions.
...
    minis_single_vesicle            boolean    Optional    For synapse models with multiple vesicles, limit spontaneous release to single vesicle when true. Default is false which allows for any number of ready vesicles to be potentially released.
    randomize_gaba_rise_time        boolean    Optional    When true, enable legacy behavior to randomize the GABA_A rise time in the helper functions. Default is false which will use a prescribed value for GABA_A rise time.
+   mechanisms                                 Optional    Properties to assign values to variables in synapse MOD files.
+                                                          The format is a dictionary with keys being the SUFFIX names of MOD files (unique names of mechanisms) and values being dictionaries of variable names in the MOD files and their values. Read about `NMODL2 SUFFIX description here <https://www.neuron.yale.edu/neuron/static/new_doc/modelspec/programmatic/mechanisms/nmodl2.html#suffix>`
```
Comment (pramodk) [inline source/sonata_simulation.rst:143]:
```suggestion
                                                          The format is a dictionary with keys being the SUFFIX names of MOD files (unique names of mechanisms) and values being dictionaries of variable names in the MOD files and their values. Read about `NMODL2 SUFFIX description here <https://nrn.readthedocs.io/en/8.2.0/hoc/modelspec/programmatic/mechanisms/nmodl2.html#suffix>`
```

By the way, just a minor thing, we can use the readthedocs link for neuron as Yale server has been having troubles for the last couple of months. And hence we are advertising/preferring readthedocs more nowadays.

File: source/sonata_simulation.rst (hunk 1 above)
Comment (WeinaJi) [reply source/sonata_simulation.rst]: Great, thanks!

**Review by jorblancoa — APPROVED** [review]

**Pull request merged**